ios/swift/AmpliSwiftSampleApp/Shared/ContentView.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ContentView.swift
+//  Shared
+//
+//  Created by Qingzhuo Zhen on 11/12/21.
+//
+
+import SwiftUI
+import CoreData
+
+struct ContentView: View {
+    @Environment(\.managedObjectContext) private var viewContext
+
+    @FetchRequest(
+        sortDescriptors: [NSSortDescriptor(keyPath: \Item.timestamp, ascending: true)],
+        animation: .default)
+    private var items: FetchedResults<Item>
+    private var ampli: Ampli = Ampli()
+    
+
+    var body: some View {
+        NavigationView {
+            List {
+                ForEach(items) { item in
+                    NavigationLink {
+                        Text("Item at \(item.timestamp!, formatter: itemFormatter)")
+                    } label: {
+                        Text(item.timestamp!, formatter: itemFormatter)
+                    }
+                }
+                .onDelete(perform: deleteItems)
+            }
+            .toolbar {
+#if os(iOS)
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    EditButton()
+                }
+#endif
+                ToolbarItem {
+                    Button(action: addItem) {
+                        Label("Add Item With Event Sent", systemImage: "plus")
+                    }
+                }
+            }
+            Text("Select an item")
+        }
+    }
+
+    private func addItem() {
+        //    Start by calling ampli.load()
```

Comment:
Friendly reminder to clean this up after we have finalized the `Ampli.instance` usage.

Reply:
Moved this to TextView now